sdk/core/core-asynciterator-polyfill/README.md

```diff
@@ -0,0 +1,54 @@
+# Azure Async Iterator Polyfill client library for JS
+
+[![Build Status](https://dev.azure.com/azure-public/adx/_apis/build/status/public.Azure.ms-rest-js)](https://dev.azure.com/azure-public/adx/_build/latest?definitionId=6)
+
+This library acts to polyfill Node 8/IE support for Symbol.asyncIterator
```

Comment:
It's also for Edge (and is insufficient for IE11 on its own, I believe). Maybe listing the platforms it is used for is more likely to be confusing than helpful?